source/code-generation/query-converter/convert-stored-procedures.txt

```diff
@@ -10,3 +10,277 @@ Convert Stored Procedures
    :depth: 1
    :class: singlecol
 
+You can import and convert your SQL stored procedures to MongoDB syntax 
+with the query converter. The query converter considers the mapping 
+rules and schema transformations defined in your project when converting
+your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
```

Comment:
Sorry I know this was in the previous PR but I think we should review this text:
> Consider SQL complexity when using the query converter. The query converter cannot convert highly complex SQL syntax. If your SQL code isn't converting accurately, refactor it and try again.

The capabilities of the feature will change over time, and we aspire to be able to convert any code regardless of its complexity. If something doesn't work, "refactoring" is unlikely to be practical. Can we make this a bit more vague, along the lines of:
> The query converter uses AI technology and is not always perfect, especially for long or complex queries or stored procedures. Some queries may not be converted correctly, or they may not be converted at all.